resources/views/welcome.blade.php

```diff
@@ -75,8 +85,12 @@
                     Blink {{ config('app.name') }} Light
                 </div>
 
-                @if (session()->has('message'))
-                <p>{{ session('message') }}</p>
+                @if (session()->has('success'))
+                <p class="success">{{ session('success') }}</p>
```

Comment:
could you indent one line inside the control blocks? and maybe make these success-message and error-message?